Userland/Libraries/LibWeb/Layout/FormattingContext.cpp

```diff
@@ -1001,9 +1001,8 @@ void FormattingContext::layout_absolutely_positioned_element(Box const& box, Ava
             - box_state.border_box_bottom();
         used_offset.set_y(height_of_containing_block + y_offset - box_state.content_height() - box_state.margin_bottom);
     } else {
-        float y_offset = box_state.margin_box_top()
-            + static_position.y();
-        used_offset.set_y(y_offset);
+        // NOTE: static position includes margin_top, border_top and padding_top for y but not for x
```

Comment:
Instead of this, how about we make the static position X and Y values both be derived from the content box instead of the mix we have now?

Reply:
thanks, actually `calculate_static_position` already returns content box position for y because `compute_box_y_position_with_respect_to_siblings` returns content box position but it was 0 for x. so I made `calculate_static_position` to also return content box position for x.